django/db/backends/base/creation.py

```diff
@@ -148,27 +148,28 @@ def _get_test_db_name(self):
             return self.connection.settings_dict['TEST']['NAME']
         return TEST_DATABASE_PREFIX + self.connection.settings_dict['NAME']
 
+    def _execute_test_db_creation(self, cursor, parameters, keepdb=False):
+        cursor.execute("CREATE DATABASE %(dbname)s %(suffix)s" % parameters)
+
     def _create_test_db(self, verbosity, autoclobber, keepdb=False):
         """
         Internal implementation - creates the test db tables.
         """
-        suffix = self.sql_table_creation_suffix()
-
         test_database_name = self._get_test_db_name()
-
-        qn = self.connection.ops.quote_name
+        test_db_param = {
+            'dbname': self.connection.ops.quote_name(test_database_name),
+            'suffix': self.sql_table_creation_suffix(),
+        }
 
         # Create the test database and connect to it.
         with self._nodb_connection.cursor() as cursor:
             try:
-                cursor.execute(
-                    "CREATE DATABASE %s %s" % (qn(test_database_name), suffix))
+                self._execute_test_db_creation(cursor, test_db_param, keepdb)
             except Exception as e:
                 # if we want to keep the db, then no need to do any of the below,
                 # just return and skip it all.
                 if keepdb:
                     return test_database_name
-
```

Comment:
To keep the diff a bit cleaner, I wouldn't make this unrelated whitespace change.